### DISCUSSION TITLE
Creating a dynamic tasks on result of dynamic task

### DISCUSSION THREAD

**Discussion by pratyush-0870** [apache/airflow#28886] [General]
Hi,

I'm spawning dynamic tasks with the below inputs
[
   {
       "1":[{},{},{}]
   },
   {
       "1":[{},{},{}]
   }
]
my first dynamic task will return list of dicts.
can I create another dynamic task to handle the return of the first dynamic task?

**Comment by Taragolis** [comment]
- [Dynamic Task Mapping: Simple mapping](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dynamic-task-mapping.html#simple-mapping)
- [Dynamic Task Mapping: Repeated Mapping](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dynamic-task-mapping.html#repeated-mapping)
